hell/adventcalendar/2025/8/index.md

```diff
@@ -1,35 +1,175 @@
 ---
 title: "What's wrong with this HTML, and is it valid?"
-author: "Your Name"
-author_bio: "Your short bio"
+author: "Patrick Brosset"
+author_bio: "I'm in love with the web platform and developer tools. I have been working on browsers for more than 10 years. Currently a product manager on the Edge team at Microsoft, I used to work at Mozilla, and was a web developer before that. I do developer relations, technical documentation, and work on a wide range of web platform technologies and tools. Always happy to chat on social media!"
 date: 2025-12-08
 author_links:
   - label: "Site"
-    url: "https://linktoyourblog123.com"
-    link_label: "linktoyourblog123.com"
+    url: "https://patrickbrosset.com"
+    link_label: "patrickbrosset.com"
+  - label: "Mastodon"
+    url: "https://mas.to/@patrickbrosset"
+    link_label: "@patrickbrosset@mas.to"
+  - label: "Bluesky"
+    url: "https://bsky.app/profile/patrickbrosset.com"
+    link_label: "@patrickbrosset.com"
 intro: "<p>Short introductory text</p>"
 image: "advent25_8"
 ---
-Some text.
-Some text.
+Behold this magnificient HTML document:
 
-Some text. Some text.
+```html
+<html>
+  <body marginheight=150 marginwidth=300 bgcolor=black text=white>
+    <marquee>
+      <b>Hello <i>HTML</b> World!</i>
+    </marquee>
+```
 
-## Heading
+To try it in your browser, copy the following line and paste it into the address bar:
 
-Some text.
+`data:text/html,<html><body marginheight=150 marginwidth=300 bgcolor=black text=white><marquee><b>Hello <i>HTML</b> World!</i></marquee>`
 
-## Heading
+## What's wrong with it?
 
-Some text.
+Everything? I mean, this HTML looks like it was written in 1998!
 
+Here's what's wrong with it:
 
-<p class="highlight"><strong>Note:</strong> Some text.</p>
+1. The document is in _quirks mode_ because it lacks a proper [`DOCTYPE` preamble](https://developer.mozilla.org/docs/Glossary/Doctype).
 
-```html
-<h1> 
-  <a href="/">
-    Hello World
-  </a>
-</h1>
-```
+   If you've never heard of quirks mode, then you're probably lucky enough to have started your web development areer after it was an important thing to know about. Suffice to say it's weird. Here are some of the ways quirks mode impacts HTML documents:
+
+   * The box model behaves differently, which affects layout and spacing.
```

Comment:
I'd love to see examples for all of these. These details may not be in the scope of the article but still super interesting. You don't have to add them to the article directly but maybe an example for each in a CodePen?